mlem/contrib/docker/base.py

```diff
@@ -335,7 +330,9 @@ class DockerEnv(MlemEnv[DockerContainer]):
     type: ClassVar = "docker"
     deploy_type: ClassVar = DockerContainer
     registry: DockerRegistry = DockerRegistry()
-    daemon: DockerDaemon = DockerDaemon(host="")
+    daemon: DockerDaemon = DockerDaemon(
+        host=os.getenv("DOCKER_HOST", default="")
```

Comment:
This should go to internals of DockerDaemon. Now it overwrites `DOCKER_HOST`, make it overwrite only if actual host was passed & restore previous env value after client is created

Reply:
removed this